### PR TITLE
Fix `rpk connect` commands

### LINKED ISSUE #8: Linting configuration command innacurate
### Issue description

The command to run the linter in this section does not work: https://docs.redpanda.com/redpanda-connect/configuration/about/#linting

`rpk connect run lint` give the error : `A maximum of one config must be specified with the run command`

The command looks like it should just be `rpk connect lint`, no `run` subcommand.

### Updates to existing documentation

https://docs.redpanda.com/redpanda-connect/configuration/about/#linting

### Link to Redpanda Slack conversation

_No response_

### PR DESCRIPTION
**Pull request by JakeSCahill**
## Description

Fixes #8

## Checks

- [ ] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [x] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/configuration/pages/using_cue.adoc [hunk 1]
```diff
@@ -27,12 +27,12 @@ ____
 CUE modules must start with a hostname. This will typically be the URL of your repository. For example: `cue mod init github.com/redpanda-data/hello-cue`.
 ____
 
-The `{rpk-command} list` command will generate a CUE package containing the types we'll need to build our configuration. Let's write this package into our project:
+The `rpk connect run list` command will generate a CUE package containing the types we'll need to build our configuration. Let's write this package into our project:
```
Comment (KasonBraley) [inline modules/configuration/pages/using_cue.adoc:30]:
```suggestion
The `rpk connect list` command will generate a CUE package containing the types we'll need to build our configuration. Let's write this package into our project:
```

File: modules/configuration/pages/about.adoc [hunk 2]
```diff
@@ -239,14 +239,14 @@ If, for example, we wanted to generate a config with a websocket input, a Kafka
 
 [,bash,subs="attributes+"]
 ----
-{rpk-command} create websocket/mapping/kafka
+rpk connect create websocket/mapping/kafka
 ----
 
-TIP: To see which components {page-component-title} offers, use `{rpk-command} list`.
+TIP: To see which components {page-component-title} offers, use `rpk connect run list`.
```
Comment (KasonBraley) [inline modules/configuration/pages/about.adoc:245]: Looks like this should have `run` removed as well.
```suggestion
TIP: To see which components {page-component-title} offers, use `rpk connect list`.
```


`$ rpk connect run list`
`Configuration file read error: list: open list: no such file or directory`


<details>
<summary>`rpk connect list`</summary>
<br>

```
Inputs:
  - amqp_0_9
  - amqp_1
  - aws_kinesis
  - aws_s3
  - aws_sqs
  - azure_blob_storage
  - azure_cosmosdb
  - azure_queue_storage
  - azure_table_storage
  - batched
  - beanstalkd
  - broker
  - cassandra
  - cockroachdb_changefeed
  - csv
  - discord
  - dynamic
  - file
  - gcp_bigquery_select
  - gcp_cloud_storage
  - gcp_pubsub
  - generate
  - hdfs
  - http_client
  - http_server
  - inproc
  - kafka
  - kafka_franz
  - mongodb
  - mqtt
  - nanomsg
  - nats
  - nats_jetstream
  - nats_kv
  - nats_stream
  - nsq
  - parquet
  - pulsar
  - read_until
  - redis_list
  - redis_pubsub
  - redis_scan
  - redis_streams
  - resource
  - sequence
  - sftp
  - socket
  - socket_server
  - sql_raw
  - sql_select
  - stdin
  - subprocess
  - twitter_search
  - websocket

Processors:
  - archive
  - avro
  - awk
  - aws_dynamodb_partiql
  - aws_lambda
  - azure_cosmosdb
  - bloblang
  - bounds_check
  - branch
  - cache
  - cached
  - catch
  - command
  - compress
  - couchbase
  - decompress
  - dedupe
  - for_each
  - gcp_bigquery_select
  - grok
  - group_by
  - group_by_value
  - http
  - insert_part
  - javascript
  - jmespath
  - jq
  - json_schema
  - log
  - mapping
  - metric
  - mongodb
  - msgpack
  - mutation
  - nats_kv
  - nats_request_reply
  - noop
  - parallel
  - parquet_decode
  - parquet_encode
  - parse_log
  - processors
  - protobuf
  - rate_limit
  - redis
  - redis_script
  - resource
  - retry
  - schema_registry_decode
  - schema_registry_encode
  - select_parts
  - sentry_capture
  - sleep
  - split
  - sql_insert
  - sql_raw
  - sql_select
  - subprocess
  - switch
  - sync_response
  - try
  - unarchive
  - wasm
  - while
  - workflow
  - xml

Outputs:
  - amqp_0_9
  - amqp_1
  - aws_dynamodb
  - aws_kinesis
  - aws_kinesis_firehose
  - aws_s3
  - aws_sns
  - aws_sqs
  - azure_blob_storage
  - azure_cosmosdb
  - azure_queue_storage
  - azure_table_storage
  - beanstalkd
  - broker
  - cache
  - cassandra
  - discord
  - drop
  - drop_on
  - dynamic
  - elasticsearch
  - fallback
  - file
  - gcp_bigquery
  - gcp_cloud_storage
  - gcp_pubsub
  - hdfs
  - http_client
  - http_server
  - inproc
  - kafka
  - kafka_franz
  - mongodb
  - mqtt
  - nanomsg
  - nats
  - nats_jetstream
  - nats_kv
  - nats_stream
  - nsq
  - opensearch
  - pulsar
  - pusher
  - redis_hash
  - redis_list
  - redis_pubsub
  - redis_streams
  - reject
  - reject_errored
  - resource
  - retry
  - sftp
  - snowflake_put
  - socket
  - splunk_hec
  - sql_insert
  - sql_raw
  - stdout
  - subprocess
  - switch
  - sync_response
  - websocket

Caches:
  - aws_dynamodb
  - aws_s3
  - couchbase
  - file
  - gcp_cloud_storage
  - lru
  - memcached
  - memory
  - mongodb
  - multilevel
  - nats_kv
  - noop
  - redis
  - ristretto
  - sql
  - ttlru

Rate Limits:
  - local
  - redis

Buffers:
  - memory
  - none
  - sqlite
  - system_window

Metrics:
  - aws_cloudwatch
  - influxdb
  - json_api
  - logger
  - none
  - prometheus
  - statsd

Tracers:
  - gcp_cloudtrace
  - jaeger
  - none
  - open_telemetry_collector

Scanners:
  - avro
  - chunker
  - csv
  - decompress
  - json_documents
  - lines
  - re_match
  - skip_bom
  - switch
  - tar
  - to_the_end

Bloblang Functions:
  - batch_index
  - batch_size
  - content
  - counter
  - deleted
  - env
  - error
  - errored
  - fake
  - file
  - file_rel
  - hostname
  - json
  - ksuid
  - metadata
  - nanoid
  - nothing
  - now
  - random_int
  - range
  - snowflake_id
  - throw
  - timestamp_unix
  - timestamp_unix_micro
  - timestamp_unix_milli
  - timestamp_unix_nano
  - tracing_id
  - tracing_span
  - ulid
  - uuid_v4
  - var

Bloblang Methods:
  - abs
  - all
  - any
  - append
  - apply
  - assign
  - bloblang
  - bool
  - bytes
  - capitalize
  - catch
  - ceil
  - collapse
  - compare_argon2
  - compare_bcrypt
  - compress
  - concat
  - contains
  - decode
  - decompress
  - decrypt_aes
  - diff
  - encode
  - encrypt_aes
  - enumerated
  - escape_html
  - escape_url_query
  - exists
  - explode
  - filepath_join
  - filepath_split
  - filter
  - find
  - find_all
  - find_all_by
  - find_by
  - flatten
  - float32
  - float64
  - floor
  - fold
  - format
  - format_json
  - format_msgpack
  - format_xml
  - format_yaml
  - from
  - from_all
  - geoip_anonymous_ip
  - geoip_asn
  - geoip_city
  - geoip_connection_type
  - geoip_country
  - geoip_domain
  - geoip_enterprise
  - geoip_isp
  - get
  - has_prefix
  - has_suffix
  - hash
  - index
  - index_of
  - int16
  - int32
  - int64
  - int8
  - join
  - json_path
  - json_schema
  - key_values
  - keys
  - length
  - log
  - log10
  - lowercase
  - map
  - map_each
  - map_each_key
  - max
  - merge
  - min
  - not
  - not_empty
  - not_null
  - number
  - or
  - parse_csv
  - parse_duration
  - parse_duration_iso8601
  - parse_form_url_encoded
  - parse_json
  - parse_jwt_es256
  - parse_jwt_es384
  - parse_jwt_es512
  - parse_jwt_hs256
  - parse_jwt_hs384
  - parse_jwt_hs512
  - parse_jwt_rs256
  - parse_jwt_rs384
  - parse_jwt_rs512
  - parse_msgpack
  - parse_parquet
  - parse_url
  - parse_xml
  - parse_yaml
  - patch
  - quote
  - re_find_all
  - re_find_all_object
  - re_find_all_submatch
  - re_find_object
  - re_match
  - re_replace
  - re_replace_all
  - replace
  - replace_all
  - replace_all_many
  - replace_many
  - reverse
  - round
  - sign_jwt_es256
  - sign_jwt_es384
  - sign_jwt_es512
  - sign_jwt_hs256
  - sign_jwt_hs384
  - sign_jwt_hs512
  - sign_jwt_rs256
  - sign_jwt_rs384
  - sign_jwt_rs512
  - slice
  - slug
  - sort
  - sort_by
  - split
  - squash
  - string
  - strip_html
  - sum
  - trim
  - trim_prefix
  - trim_suffix
  - ts_add_iso8601
  - ts_format
  - ts_parse
  - ts_round
  - ts_strftime
  - ts_strptime
  - ts_sub
  - ts_sub_iso8601
  - ts_tz
  - ts_unix
  - ts_unix_micro
  - ts_unix_milli
  - ts_unix_nano
  - type
  - uint16
  - uint32
  - uint64
  - uint8
  - unescape_html
  - unescape_url_query
  - unique
  - unquote
  - uppercase
  - values
  - with
  - without
  - zip
```
</details>

File: modules/configuration/pages/using_cue.adoc [hunk 3]
```diff
@@ -27,12 +27,12 @@ ____
...
 
 [,bash,subs="attributes+"]
 ----
 mkdir benthos
-{rpk-command} list --format cue > benthos/schema.cue
+rpk connect run list --format cue > benthos/schema.cue
```
Comment (KasonBraley) [inline modules/configuration/pages/using_cue.adoc:35]:
```suggestion
rpk connect list --format cue > benthos/schema.cue
```

File: modules/guides/pages/bloblang/about.adoc [hunk 4]
```diff
@@ -10,7 +10,7 @@ You can also execute Bloblang mappings on the command-line with the `blobl` subc
 
 [,shell]
 ----
-cat data.jsonl | {rpk-command} blobl 'foo.(bar | baz).buz'
+cat data.jsonl | rpk connect run blobl 'foo.(bar | baz).buz'
```
Comment (KasonBraley) [inline modules/guides/pages/bloblang/about.adoc:13]:
```suggestion
cat data.jsonl | rpk connect blobl 'foo.(bar | baz).buz'
```

File: modules/guides/pages/bloblang/about.adoc [hunk 5]
```diff
@@ -432,8 +432,8 @@ It's possible to execute unit tests for your Bloblang mappings using the standar
 
 == Troubleshooting
 
-. I'm seeing `unable to reference message as structured (with 'this')` when I try to run mappings with `{rpk-command} blobl`.
+. I'm seeing `unable to reference message as structured (with 'this')` when I try to run mappings with `rpk connect run blobl`.
```
Comment (KasonBraley) [inline modules/guides/pages/bloblang/about.adoc:435]:
```suggestion
. I'm seeing `unable to reference message as structured (with 'this')` when I try to run mappings with `rpk connect blobl`.
```

File: modules/guides/pages/bloblang/about.adoc [hunk 6]
```diff
@@ -432,8 +432,8 @@ It's possible to execute unit tests for your Bloblang mappings using the standar
...
 
-That particular error message means the mapping is failing to parse what's being fed in as a JSON document. Make sure that the data you are feeding in is valid JSON, and also that the documents _do not_ contain line breaks as `{rpk-command} blobl` will parse each line individually.
+That particular error message means the mapping is failing to parse what's being fed in as a JSON document. Make sure that the data you are feeding in is valid JSON, and also that the documents _do not_ contain line breaks as `rpk connect run blobl` will parse each line individually.
```
Comment (KasonBraley) [inline modules/guides/pages/bloblang/about.adoc:437]:
```suggestion
That particular error message means the mapping is failing to parse what's being fed in as a JSON document. Make sure that the data you are feeding in is valid JSON, and also that the documents _do not_ contain line breaks as `rpk connect blobl` will parse each line individually.
```

File: modules/guides/pages/bloblang/walkthrough.adoc [hunk 7]
```diff
@@ -4,7 +4,7 @@
 
 Bloblang is the most advanced mapping language that you'll learn from this walkthrough (probably). It is designed for readability, the power to shape even the most outrageous input documents, and to easily make erratic schemas bend to your will. Bloblang is the native mapping language of {page-component-title}, but it has been designed as a general purpose technology ready to be adopted by other tools.
 
-In this walkthrough you'll learn how to make new friends by mapping their documents, and lose old friends as they grow jealous and bitter of your mapping abilities. There are a few ways to execute Bloblang but the way we'll do it in this guide is to pull a {page-component-title} docker image and run the command `{rpk-command} blobl server`, which opens up an interactive Bloblang editor:
+In this walkthrough you'll learn how to make new friends by mapping their documents, and lose old friends as they grow jealous and bitter of your mapping abilities. There are a few ways to execute Bloblang but the way we'll do it in this guide is to pull a {page-component-title} docker image and run the command `rpk connect run blobl server`, which opens up an interactive Bloblang editor:
```
Comment (KasonBraley) [inline modules/guides/pages/bloblang/walkthrough.adoc:7]:
```suggestion
In this walkthrough you'll learn how to make new friends by mapping their documents, and lose old friends as they grow jealous and bitter of your mapping abilities. There are a few ways to execute Bloblang but the way we'll do it in this guide is to pull a {page-component-title} docker image and run the command `rpk connect blobl server`, which opens up an interactive Bloblang editor:
```

File: modules/guides/pages/bloblang/walkthrough.adoc [hunk 8]
```diff
@@ -759,7 +759,7 @@ tests:
 
 As you can see we've defined a single test, where we point to our mapping file which will be executed in our test. We then specify an input message which is a reduced version of the document we tried out before, and finally we specify output predicates, which is a JSON comparison against the output document.
 
-We can execute these tests with `{rpk-command} test ./naughty_man_test.yaml`, {page-component-title} will also automatically find our tests if you simply run `{rpk-command} test ./...`. You should see an output something like:
+We can execute these tests with `rpk connect run test ./naughty_man_test.yaml`, {page-component-title} will also automatically find our tests if you simply run `rpk connect run test ./...`. You should see an output something like:
```
Comment (KasonBraley) [inline modules/guides/pages/bloblang/walkthrough.adoc:762]:
```suggestion
We can execute these tests with `rpk connect test ./naughty_man_test.yaml`, {page-component-title} will also automatically find our tests if you simply run `rpk connect test ./...`. You should see an output something like:
```

File: modules/guides/pages/streams_mode/about.adoc [hunk 9]
```diff
@@ -36,7 +36,7 @@ output:
   sync_response: {}
 ----
 
-Will register an endpoint `/meow`, which will be prefixed with the name `foo` to become `/foo/meow`. This behavior is intended to make a clearer distinction between endpoints registered by different streams, and prevent collisions of those endpoints. However, you can disable this behavior by setting the flag `--prefix-stream-endpoints` to `false` (`{rpk-command} streams --prefix-stream-endpoints=false ./streams/*.yaml`).
+Will register an endpoint `/meow`, which will be prefixed with the name `foo` to become `/foo/meow`. This behavior is intended to make a clearer distinction between endpoints registered by different streams, and prevent collisions of those endpoints. However, you can disable this behavior by setting the flag `--prefix-stream-endpoints` to `false` (`rpk connect run streams --prefix-stream-endpoints=false ./streams/*.yaml`).
```
Comment (KasonBraley) [inline modules/guides/pages/streams_mode/about.adoc:39]:
```suggestion
Will register an endpoint `/meow`, which will be prefixed with the name `foo` to become `/foo/meow`. This behavior is intended to make a clearer distinction between endpoints registered by different streams, and prevent collisions of those endpoints. However, you can disable this behavior by setting the flag `--prefix-stream-endpoints` to `false` (`rpk connect streams --prefix-stream-endpoints=false ./streams/*.yaml`).
```

**Review by KasonBraley — COMMENTED** [review]
I commented on more that I found while reviewing. There could be more that I missed. I just used the main help as the reference to list all the commands that are not subcommands of `run`.

```bash
rpk connect --help               
NAME:
   redpanda-connect - A stream processor for mundane tasks - https://docs.redpanda.com/redpanda-connect

USAGE:
   redpanda-connect [global options] command [command options] 

DESCRIPTION:
   Either run Redpanda Connect as a stream processor or choose a command:

     redpanda-connect list inputs
     redpanda-connect create kafka//file > ./config.yaml
     redpanda-connect -c ./config.yaml
     redpanda-connect -r "./production/*.yaml" -c ./config.yaml

COMMANDS:
   echo      Parse a config file and echo back a normalised version
   lint      Parse Redpanda Connect configs and report any linting errors
   run       Run Redpanda Connect in normal mode against a specified config file
   streams   Run Redpanda Connect in streams mode
   list      List all Redpanda Connect component types
   create    Create a new Redpanda Connect config
   test      Execute Redpanda Connect unit tests
   template  Interact and generate Redpanda Connect templates
   blobl     Execute a Redpanda Connect mapping on documents consumed via stdin
   studio    Interact with Benthos studio (https://studio.benthos.dev)
   help, h   Shows a list of commands or help for one command
```

**Review by r-vasquez — APPROVED** [review]

**Review by micheleRP — APPROVED** [review]
lgtm

**Pull request merged**